CONTRIBUTING.md

```diff
@@ -23,10 +23,16 @@ Within Github, navigate to <https://github.com/python/mypy> and fork the reposit
 #### (2) Clone the mypy repository and enter into it
 
 ```bash
-git clone git@github.com:<your_username>/mypy.git
+git clone https://github.com/<your_username>/mypy.git
```

Comment:
I don't think this is a good idea given that this is the contributing guide where people would want to push to the repository but cannot with a https clone. Maybe add the link to GitHub shh docs but keep this command unchanged?

Reply:
@hamdanal No problem at all. I'm just still a newbie about github. If I'm to amend this pull request should I do it locally and then push onto my fork of mypy for the pull request to automatically amend or just cancel this pull request and create a new one?

Reply:
@ag-tafe The former. You can update this PR by making changes locally and pushing to the branch in your fork. Any new commits will appear in the PR history.

In case helpful as a reference, this is what the cloning instructions look like in CPython's contribution guide: https://devguide.python.org/getting-started/setup-building/